jujucrashdump/crashdump.py

```diff
@@ -155,6 +162,13 @@ def juju_storage_pools():
     juju_cmd('storage-pools --format=yaml', to_file='storage_pools.yaml')
 
 
+def run_ssh(host, timeout, ssh_cmd, cmd):
+    for ip in host:
```

Comment:
Why are there multiple ips in host? Why do we break on success? Not sure what's going on here

Reply:
Each unit can have several interfaces and IP addresses. This cycles through them and finds the first working.
I will add a comment there